src/app/components/pages/IsaacCompetition/EntryForm/CompetitionEntryForm.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { Form, Row, Col, Button, Container, FormGroup, Label, Input } from "reactstrap";
+import { InputType } from "reactstrap/es/Input";
+
+interface CompetitionEntryFormProps {
+  handleTermsClick: (event: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => void;
+}
+
+const renderFormGroup = (label: string, type: string, id: string, options: string[] = []) => (
+  <FormGroup>
+    <Label className="entry-form-sub-title">{label}</Label>
+    {type === "select" ? (
+      <Input type="select" id={id}>
+        {options?.length > 0 &&
+          options.map((option, index) => (
+            <option key={`${option}-${index}`} value={option === "Please select from the list" ? "" : option}>
+              {option}
+            </option>
+          ))}
+      </Input>
+    ) : (
+      <Input type={type as InputType} id={id} />
+    )}
+  </FormGroup>
+);
+
+const CompetitionEntryForm = ({ handleTermsClick }: CompetitionEntryFormProps) => {
+  const handleSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+  };
+
+  return (
+    <div className="py-5">
+      <div className="entry-form-background-img entry-form-section">
+        <Container className="pb-2">
+          <Form onSubmit={handleSubmit}>
+            <h1 className="py-4 entry-form-title">Enter the competition</h1>
+            <Row className="d-flex flex-column flex-md-row">
+              <Col lg={6}>
+                {renderFormGroup("Name", "text", "formSubtitle1")}
+                {renderFormGroup("School", "text", "formSubtitle2")}
+                {renderFormGroup("Email", "text", "formSubtitle3")}
+              </Col>
+              <Col lg={6}>
+                {renderFormGroup("Link to submission", "text", "formSubtitle4")}
+                {renderFormGroup("Group", "select", "formSubtitle5", [
+                  "Please select from the list",
+                  "Option 1",
```

Comment:
Please clarify the specific options for this placeholders. If the options are not currently documented, we should consult with Vicki to define and specify them :)